### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1041] [Galaxy Trivia] [answered]
```
1  4  8  13  21  30  36 ?
```

**Comment by peterbaccaro** [comment, marked as answer]
The next number in the sequence is **45**.

This is because each number is calculated as the previous number plus the number of letters when the previous number is spelt.

The last number is 36 so "thirty six" has 9 letters.
Therefore, 36 + 9 = 45.


Each term increases by the number of letters in the previous term (ignoring any hyphens):

**Comment by CelioAmaral** [comment]
The Logic of the series is as follows :
1 + one (3 letters) = 4
4 + four (4 letters) = 8
8 + eight (5 letters) = 13
13 + thirteen (8 letters) = 21
21 + twenty one (9 letters) = 30
30 + thirty (6 letters) = 36
36 + thirty six (9 letters) = 45

So answer is 45
